core/src/com/ericc/the/game/MainGame.java

```diff
@@ -16,10 +16,7 @@
 import com.ericc.the.game.map.Generator;
 import com.ericc.the.game.map.Map;
 import com.ericc.the.game.systems.logic.*;
-import com.ericc.the.game.systems.realtime.ScreenBoundariesGetterSystem;
-import com.ericc.the.game.systems.realtime.AnimationSystem;
-import com.ericc.the.game.systems.realtime.RenderSystem;
-import com.ericc.the.game.systems.realtime.TileChanger;
+import com.ericc.the.game.systems.realtime.*;
```

Comment:
Skoro potrzebujemy wszystkich systemów to równie dobrze można importować com.ericc.the.game.systems.*

Reply:
nie wiem czy to jest słuszna uwaga, najlepiej niech importowaniem zajmuje się Intellij. On sam sobie wykminia kiedy dać *.

Reply:
A, dobra, zrozumiałem o co cho, ale to w takim razie niestety nie jest w ogóle słuszna uwaga, bo gdyby napisać import com.ericc.the.game.systems.* to w miejscach użycia systemów trzeba będzie pisać realtime.System albo logic.System.